pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
```

Comment:
I think we should add a minimal version of setuptools as a build requirement to ensure that a compatible version of setuptools is installed.

```suggestion
requires = ["setuptools>=61", "setuptools-scm"]
```

Reply:
Agreed